dev-docs/bidders/screencore.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Screencore
+description: Prebid Screencore Bidder Adaptor
+biddercode: screencore
+gdpr_supported: true
```

Comment:
please change this to `tcfeu_supported as described in https://github.com/prebid/prebid.github.io/pull/4725